### DISCUSSION TITLE
Software signed by "Tonalio GmbH"

### DISCUSSION THREAD

**Discussion by Alex210688** [sandboxie-plus/Sandboxie#3526] [Q&A Feedback]
Is the certificate legit? It is signed by  "Tonalio GmbH"

**Comment by isaak654** [comment]
It is an Austrian company owned by a @DavidXanatos friend who is responsible of signing the Sandboxie versions marked as stable (see [this explanation](https://github.com/sandboxie-plus/Sandboxie/issues/164#issuecomment-716055266)). For some reason, he asked to remain anonymous in order not to appear on the [organization members](https://github.com/orgs/sandboxie-plus/people) list, but the company's references are publicly available online.

**Reply by bastik-1001** [reply]
Should we mention the name of the signee in the main readme.md in the download section?

Maybe remind David to mention the signee on the download page on sandboxie-plus.com?

**Reply by isaak654** [reply]
> Should we mention the name of the signee in the main readme.md in the download section?

Personally, I'm indifferent. However, feel free to create a pull request for this.

**Reply by bastik-1001** [reply]
I was "hoping" David would say if he approves of it. So far this support question is the only time, I am aware, someone asked. It still could not hurt to name the official signee, but one might feel different.
